.testcaferc.js

```diff
@@ -95,7 +110,7 @@ const config = {
   }),
 
   // limit concurrency when running flaky tests
-  concurrency: OKTA_SIW_ONLY_FLAKY ? 1 : undefined,
+  concurrency: OKTA_SIW_ONLY_FLAKY || !env.CHROME_HEADLESS ? 1 : (env.OKTA_SIW_EN_LEAKS ? 2 : 4),
```

Comment:
little confusing to read.
```suggestion
  // OKTA_SIW_ONLY_FLAKY: 1 ← flaky
  // !CHROME_HEADLESS: 1 ← for debugging tests
  // OKTA_SIW_EN_LEAKS: 2 ← somewhat flaky
  // !OKTA_SIW_EN_LEAKS: 4 ← most tests
  concurrency: (OKTA_SIW_ONLY_FLAKY || !env.CHROME_HEADLESS) 
    ? 1 
    : (env.OKTA_SIW_EN_LEAKS ? 2 : 4),
```